provider/godaddy/godaddy.go

```diff
@@ -506,3 +546,13 @@ func maxOf(vars ...int64) int64 {
 
 	return max
 }
+
+func toString(obj interface{}) string {
```

Comment:
optional, maybe this also should return the error as well instead of wrapping it into the string

Reply:
This function is used to convert known structure to json representation string to log it in this usage fmt.Sprintf("....%s...", toString(myStruct)) and dual valued is not allowed